lib/options-manager.js

```diff
@@ -1,11 +1,11 @@
 'use strict';
-const os = require('os');
```

Comment:
Shouldn't this import statement stays as `os` is used on line 32?

Reply:
Yes, sorry I missed this, apparently I forgot to rerun 'npm test' after I re-added the directory used when xo is run from outside a project.

Also: I'm just noticing it doesn't appear this repo runs CI on PR's.